cpp/oneapi/dal/algo/decision_forest/backend/gpu/train_misc_structs.hpp

```diff
@@ -137,6 +137,8 @@ struct train_context {
     bool use_private_mem_buf_ = true; // valuable for classification only
         // for switching between private mem and other buffers(local, global) for storing class hist
 
+    splitter_mode splitter_mode_value;
+    std::uint64_t seed;
```

Comment:
Shouldn't it be `splitter_mode_value_` and `seed_`? To be aligned with other variables' names.